portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,90 @@
+/* Display reset */
+html,body,div,span,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,abbr,address,cite,code,del,dfn,em,img,ins,kbd,q,samp,small,strong,sub,sup,var,b,i,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,figcaption,figure,footer,header,hgroup,menu,nav,section,summary,time,mark,audio,video {
+border:0;
+font:inherit;
+font-size:100%;
+margin:0;
+padding:0;
+vertical-align:baseline;
+}
+
+article,aside,details,figcaption,figure,footer,header,hgroup,menu,nav,section {
+display:block;
+}
+
+html,body {
+    background-color: #421C52; 
+    font-family: Calibri, Candara, Segoe, "Segoe UI", Optima, Arial, sans-serif;
+    font-size: 17px;
+    font-style: normal;
+    font-weight: 200;
+    line-height: 18.3px;
+    color: white;
+}
+
+.clear{
+    clear: both;
+}
+
+/* Text format */
+h1 { 
+    margin-top: 60px;  
+    text-align: center;
+    font-size: 28pt;
+    font-style: bold;
+    font-weight: 700;
+    line-height: 30.4px;
+}
+
+h2 {
+    font-size: 23px; 
+    font-style: normal; 
+    font-weight: 700; 
+    line-height: 24.4px; 
+}
+
+
+/* ID format */
+#title {
+    font-style: italic;
+    font-size: 15px;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 87%;
+    max-width: 800px;
+    background-color: #732C7B;
+}
+
+#left-details {
+    padding: 20px;
+    float: left;
+}
+
+#right-details {
+    padding: 20px;
+    float: right;
+}
+
+#display-container {
+    padding: 20px 30px;
+    margin: 0;
 }
 
 #greeting-container {
-  margin-top: 20px;
+    padding: 20px 30px;
+    margin-left: auto;
+    margin-right: auto;
 }
+
+/* Class format */
+.details {
+    padding: 25px 35px;
+    border-bottom: #9C8AA5;
+}
+
+.area {
+    padding: 0 40px;
+}
```

Comment:
It's good to have a new-line for the last line because it's unusual to not have one.

Reply:
The new-line at the end is added.